packages/next/src/server/app-render/app-render-prerender-utils.ts

```diff
@@ -31,168 +31,6 @@ export function prerenderAndAbortInSequentialTasks<R>(
   }
 }
 
-const PENDING = 0
-const COMPLETE = 1
-const INTERRUPTED = 2
-const ERRORED = 3
-
-export class ServerPrerenderStreamResult {
```

Comment:
This class had a rule violation, but it turned out that it was unused. It's last usage was removed in #79414.